docs/private-networks/concepts/node-sync-private.md

```diff
@@ -24,9 +24,9 @@ Select the sync mode based on your network's requirements and node purposes.
 
 | Sync mode                                                                            | Description                                                                                                                                                                         | Requirements | Limitations |
 |--------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|-------------|
-| [Snap](../../public-networks/concepts/node-sync.md#snap-synchronization)             | Recommended for fastest sync and lowest storage requirements on Mainnet. Downloads as many leaves of the trie as possible and reconstructs the trie locally. Faster than fast sync. | Available as an _early access feature_ in Besu version 24.7.1 or later | Cannot switch from fast sync to snap sync mid-process. |
+| [Snap](../../public-networks/concepts/node-sync.md#snap-synchronization)             | Recommended for fastest sync and lowest storage requirements on Mainnet. Downloads as many leaves of the trie as possible and reconstructs the trie locally. Faster than fast sync. | Besu version 24.3.0 or later | Cannot switch from fast sync to snap sync mid-process. |
 | [Checkpoint](../../public-networks/concepts/node-sync.md#checkpoint-synchronization) | Syncs from a specific checkpoint block configured in the genesis file. Fastest sync mode with lowest storage requirements.                                                          | Besu version 22.4.3 or later | Not supported for QBFT or IBFT 2.0 networks without a checkpoint configuration. |
-| [Fast](../../public-networks/concepts/node-sync.md#fast-synchronization-deprecated)             | Downloads block headers and transaction receipts, verifies chain from genesis block.                                                                                                | None | Not supported with private transactions. |
+| [Fast](../../public-networks/concepts/node-sync.md#fast-synchronization-deprecated)             | Downloads block headers and transaction receipts, verifies chain from genesis block.                                                                                                | None | Deprecated. Not supported with private transactions. |
 | [Full](../../public-networks/concepts/node-sync.md#full-synchronization)             | Default for all private networks. Downloads and verifies the entire blockchain and state from genesis block, building an archive node with full state history.                      | None | Slowest sync mode, requires the most disk space. |
 
 :::warning Early access feature 
```

Comment:
Should we update this note, too, and other places where we mention the early access feature? Or we can address in the separate PR (https://github.com/hyperledger/besu-docs/pull/1739)

Reply:
I think that can be a separate PR